docs/UserGuide.md

```diff
@@ -697,6 +728,13 @@ Staff-Snap applicant data are saved automatically as a [JSON](#glossary) file `[
 **Q**: Is my data secure? <br>
 **A**: Yes, your data is stored locally in your computer and is not accessible by anyone else.
 
+**Q**: Will I lose my data when I exit the app?
+**A**: No you will not. The data is stored on your computer and can still be displayed after exiting the app.
```

Comment:
Markdown line break is missing here

Reply:
Thanks for the review Craig, does it look better now?